store/src/main/java/in/testpress/store/models/CouponCodeResponse.kt

```diff
@@ -0,0 +1,68 @@
+package `in`.testpress.store.models
+
+import com.google.gson.annotations.SerializedName
+
+data class CouponCodeResponse(
+        val date: String? = null,
+        val voucher: Voucher? = null,
+        val checksum: String? = null,
+        val id: Int? = null,
+        val email: String? = null,
+        val zip: String? = null,
+        val amount: String? = null,
+        val apikey: String? = null,
+        val phone: String? = null,
+        val name: String? = null,
+        val user: Int? = null,
+        val status: String? = null,
+
+        @field:SerializedName("amount_without_discounts")
```

Comment:
is serializedName necessary?

Reply:
Please check if it is working without it

Reply:
It won't work if we remove the serialized. the variable name and the son variable name should be similar or else it won't recognize. Here the variable name is not in camelCases so the Serialized name is needed.